configs/nncf_compression/wider_face/face-detection-0200-config_int8.py

```diff
@@ -0,0 +1,176 @@
+# model settings
```

Comment:
should not it be in training_extensions?

Reply:
Yes, it should!

Reply:
Fixed